src/data/selector.ts

```diff
@@ -111,8 +111,8 @@ export interface DateTimeSelector {
 export interface DeviceSelector {
   device: {
     integration?: string;
-    manufacturer?: string;
-    model?: string;
+    manufacturer?: string | [string];
+    model?: string | [string];
```

Comment:
IMHO, this should not be added to these keys.

For example, this can create conflicts between manufacturers and models.

Maybe a better solution would be to allow for multiple `device` definitions instead?

Reply:
Certainly most of the time there will be one manufacturer, and multiple models. However, there are cases where identical (or interchangeable) hardware is made from multiple manufacturers (or the name changes). I expect it will be used sparingly, but could be quite useful in that case. Chances of a name clash are pretty low I would think.